src/js/view/popup/scheduleCreationPopup.js

```diff
@@ -295,7 +295,8 @@ ScheduleCreationPopup.prototype._onClickSaveSchedule = function(target) {
                 isAllDay: isAllDay,
                 state: state.innerText,
                 triggerEventName: 'click',
-                id: this._schedule.id
+                id: this._schedule.id,
+                _feId: this._schedule.__fe_id
```

Comment:
Schedule.prototype.cid / ScheduleViewModel.prototype.cid

Reply:
@dongsik-yoo 
If the user does not use the default pop-up UI, or if the user directly calls the `updateSchedule` method, `cid` does not exist.
So, to find the schedule in the schedule collection, I modified it to pass the calendar Id (`originCalendarId`) of the schedule before the change.